cg/meta/orders/api.py

```diff
@@ -173,8 +173,15 @@ def _add_customer_to_message(self, order, message):
             message += f", {customer_name} ({customer_id})"
         return message
 
+    def _submit_fluffy(self, order: dict) -> dict:
+        """Submit a batch of ready made libraries for FLUFFY analysis."""
+        return self._submit_pools(order)
+
     def _submit_rml(self, order: dict) -> dict:
-        """Submit a batch of ready made libraries."""
+        """Submit a batch of ready made libraries for sequencing."""
+        return self._submit_pools(order)
```

Comment:
What is the benefit of splitting submit rml and submit fluffy off pools? Seems they just pass the method call forward without any unique handling

Reply:
yeah, it's a dynamic call to the method, I guess we could have solved it by using the get-method as well